fluss-server/src/test/java/com/alibaba/fluss/server/coordinator/CoordinatorEventProcessorTest.java

```diff
@@ -761,6 +766,59 @@ void testNotifyOffsetsWithShrinkISR(@TempDir Path tempDir) throws Exception {
         verifyReceiveRequestExceptFor(3, leader, NotifyKvSnapshotOffsetRequest.class);
     }
 
+    @Test
+    void testProcessAdjustIsr() throws Exception {
```

Comment:
This test seems doing a void testing. It gets the current `LeaderAndIsr`s and update `LeaderAndIsr`s without any changes, and then verify the `LeaderAndIsr` again. If the `processAdjustIsr` has something wrong, this test can't find anything. 

I suggest to do some modification on the retrieved `LeaderAndIsr` from ctx, like changing the leader to the second element in isr, and then send the new `LeaderAndIsr` within `AdjustIsrReceivedEvent` to coordinator, finally, asserting the `LeaderAndIsr` in ctx and response is updated to the new leader.

Reply:
Sorry, I didn't notice the `bucketEpoch + 1` in the `newLeaderAndIsr()`.